chef/cookbooks/neutron/templates/default/dnsmasq-neutron.conf.erb

```diff
@@ -0,0 +1,4 @@
+# this file is managed by chef to give extra options to dnsmasq
+<% if @low_mtu -%>
+dhcp-option-force=26,1458
```

Comment:
So I guess it'd be useful to know how you reached 1458 to decide. I'm not sure where the 42 overhead comes from (except for the experimental approach where you try it). GRE can be 4, 8 or 12 bytes, IPv4 is 20 bytes, IPv6 (assuming we switch to IPv6 later) would be 40 bytes instead of 20 bytes.